glenn_utilities/src/glenn_utilities/ros_bagger.py

```diff
@@ -15,10 +17,27 @@ def __init__(self):
         today = datetime.datetime.now().strftime("%Y_%m_%d")
         now = datetime.datetime.now().strftime("%Y_%m_%d_%H_%M_%S")
 
-        bag_dir = os.path.expandvars("$HOME/glenn_bags" + "/" + today)
+        # default bag location
+        desired_path="$HOME" 
+
+        # Will now store bags with respect to ROS_LOG_DIR if it has been set
+        # Bags to external USB stick if the ROS_LOG_DIR is set to the path of the stick
+        if 'ROS_LOG_DIR' in os.environ:
+            desired_path =os.getenv('ROS_LOG_DIR')
+
+            #gets parent directory of ros logs
+            desired_path= os.path.abspath(os.path.join(desired_path , "../"))
```

Comment:
Is there a particular reason to take in the input path and add `..` to it? Can the input path be that path ahead of time?

Reply:
I updated the path to be cleaner.